### DISCUSSION TITLE
Priority roads

### DISCUSSION THREAD

**Discussion by PG1966** [gama-platform/gama#92] [Q&A] [answered]
Good morning,

how you can introduce the priority_roads parameter.

Thank you in advance

**Comment by minhduc0711** [comment, marked as answer]
Hello, I assume that you are talking about the driving skill. You can simply use the `priority_roads` attribute of the `skill_road_node`, which expects a list of roads:
```
some_node.priority_roads <- [some_road_1, some_road_2];
```

**Reply by PG1966** [reply]
Thank so much
